modules/nw-operator-cr.adoc

```diff
@@ -311,6 +311,7 @@ One of the following additional audit log targets:
 |====
 // end::policy-audit[]
 
+[id="gatewayConfig-object_{context}"]
```

Comment:
There might be several instances of OpenShift that might need to be changed to `{product-title}` For example, does OpenShift SDN need to be {product-title} SDN? I did notice this is several other modules, so I was not sure if the feature name needed the variable, but at least wanted to mention it.

Reply:
This is the name of a container network interface, one of two that we provide. I believe the CNI name is the same regardless of what platform you use it on. Will double check with other networking folks.

Reply:
Definitely `OpenShift SDN`. STET.